movie_api/urls.py

```diff
@@ -0,0 +1,23 @@
+"""movie_api URL Configuration
```

Comment:
The project name in the docstring is incorrectly referred to as 'movie_api'. It should be 'cinema' to match the correct project name.